docs/proposals/volume-expansion.md

```diff
@@ -0,0 +1,95 @@
+# Proposal: Volume Expansion
+
+Last updated: 09.02.21
+
+## Abstract
+
+This proposal contains design and implementation considerations about volume expansion pf LVG volumes
+
+## Background
+
+Currently CSI doesn't support volume expansion feature, this proposal aims to solve this problem for LVG volumes.
+
+## Proposal
+
+There are two requests in CSI spec, that can be implemented: `ControllerExpandVolume` and `NodeExpandVolume`. Also there are 
+2 types of Volume Expansion: `ONLINE` and `OFFLINE`. The differences is that online volume expansion indicates that volumes may be expanded
+ when published to a node. `NodeExpandVolume` always perform online volume expansion, so `NodeExpandVolume` must be called after NodeStage request and may be called 
+after NodePublishVolume. `ControllerExpandVolume` can perform both type of volume expansion. In case of offline volume expansion
+`ControllerExpandVolume` must be called after ControllerPublishVolume. If volume is published, `ControllerExpandVolume` must be called
```

Comment:
Should it be the _ONLINE_ case after ControllerPublishVolume ?

Reply:
This statement is not really correct, I'll remove it. Thanks!